code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -111,7 +111,7 @@
 	name = "chocolate donut"
 	desc = "Goes great with a glass of warm milk."
 	icon_state = "donut_choc"
-	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1) //the coco reagent is just bitter.
+	bonus_reagents = list(/datum/reagent/consumable/hot_coco = 3, /datum/reagent/consumable/sprinkles = 1) //the cocoa reagent is just bitter.
```

Comment:
Did you not change hot cocoa, but the comment?

Reply:
Really odd that these didn't change. I'll fix it shortly.